tabbycat/standings/tests/test_standings.py

```diff
@@ -253,13 +252,15 @@ def setUp(self):
         debate = self.set_up_ignorable_debate()
         debate.ballotsubmission_set.update(confirmed=False)
 
-    @expectedFailure
     def test_draw_strength(self):
-        super().test_draw_strength()
+        # The ignorable debate still counts as an opponent (because they definitely faced each
+        # other) but does not count as a win (because it's unconfirmed). So the losing team has
+        # faced the winning team 3 times (including ignorable debate), and the winning team has 2
+        # wins (excluding ignorable debate), for a losing team draw strength of 3 * 2 = 6.
+        self._base_metric_test({'draw_strength': [0, 6]})
 
-    @expectedFailure
     def test_draw_strength_speaks(self):
-        super().test_draw_strength_speaks()
+        self._base_metric_test({'draw_strength_speaks': [591, 609]})
```

Comment:
I'd include a comment here too (even if it's just a reference to the above one, like "Similar logic to `test_draw_strength` above".